src/ngSanitize/sanitize.js

```diff
@@ -262,11 +236,34 @@ function makeMap(str, lowercaseKeys) {
   return obj;
 }
 
+var baseNode;
+(function(window) {
+  var doc;
+  if (window.DOMDocument) {
+    doc = new window.DOMDocument();
```

Comment:
That is my understanding, and also I was able to verify this imperatively. If you try to create `<img src="javascript:alert(1)">`  in regular document you can see either the alert or that it was canceled in the network tab. Creating the same thing on the new document does nothing.